src/main.js

```diff
@@ -322,17 +322,29 @@ class MiniGraphCard extends LitElement {
     `;
   }
 
+  renderLoader() {
+    return html`
+        <div style="display: flex;">
```

Comment:
Do we need this div? I think, it should work without.

Reply:
I do not think `div` is needed

Reply:
Indeed removed